### PR TITLE
Add a guide for custom CSS

### PR DESCRIPTION
**Pull request by astorije**
Extracted from #84.

### CODE REVIEW

**Comment by astorije** [comment]
I don't see why not. This should end up becoming external packages over time, but we'll have more control in saying "no thanks, please create a package instead" in the doc rather than in the publicly-editable wiki.

**Comment by xPaw** [comment]
I think it should just simply stay in the repo wiki, instead of moving it here as a intermediary step.

File: _guides/custom-css.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+Towards the bottom of the **Settings** page, you will find a **Custom Stylesheet** field:
+
+![Custom Stylesheet textarea which can be found in the Settings page](/img/custom-stylesheet-textarea.png){:width="468px"}
+
+Simply add your custom CSS, and the modifications will be applied when clicking outside the field.
+
+Note that The Lounge evolves rather quickly, and new versions often change the existing HTML and CSS files. This causes custom CSS snippets to often require changes as new versions get released. Use your [browser's developer tools](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools) when your trying to add or edit custom CSS.
+
+{: .alert.alert-warning role="alert"}
+Settings are not synced across devices at the moment, so any custom CSS you set will only apply to this specific instance of The Lounge.
```
Comment (xPaw) [inline _guides/custom-css.md]: They are though?

Comment (astorije) [reply _guides/custom-css.md]: Ah, that was written before that feature I think :)

Comment (astorije) [reply _guides/custom-css.md]: Removed.

**Comment by astorije** [comment]
Also added a commit to fix images:

Before | After
--- | ---
screen shot 2018-11-03 at 15 02 48 | screen shot 2018-11-03 at 15 03 06

**Pull request merged**